packages/framework/src/Model/Security/LoginService.php

```diff
@@ -70,5 +70,8 @@ public function loginUser(User $user, Request $request)
         // dispatch the login event
         $event = new InteractiveLoginEvent($request, $token);
         $this->traceableEventDispatcher->dispatch(SecurityEvents::INTERACTIVE_LOGIN, $event);
+
+        //migrate session
```

Comment:
This comment is not relevant. You can see that the session will migrate in the code and the reason why it happens should be described in the commit message (and that will be handled when this PR is merged).

Reply:
Hi @dzarda86, thank you for your time with this PR. 
I removed the comment, so we can merge and close issue and so no further action from you is required. 

We're looking forward to your contributions. ✨